hack/conformance.sh

```diff
@@ -2,11 +2,17 @@
 
 set -xeuo pipefail
 
+SCRIPT_PATH=$(dirname "$(realpath "$0")")
+
 ARTIFACTS=${ARTIFACTS:-${PWD}}
+
 config_file=${1:-}
 sonobuoy_version=0.50.0
 [[ -f "$config_file" ]] && sonobuoy_version=$(jq -r '.Version' "$config_file" | grep -oE '[0-9\.]+')
 
+conformance_image_config_file="$SCRIPT_PATH/conformance-config-file.yaml"
+! [[ -f "$conformance_image_config_file" ]] && echo "FATAL: Conformance image config file does not exists" &>2 && return 1
```

Comment:
```
hack/conformance.sh: line 14: return: can only `return' from a function or sourced script
```

Can it be that you wanted to write `exit 0`?

Reply:
Yes, sorry I will fix it :see_no_evil: